Web3-API/web3hub
# Ens subdomains registration

This PR includes necessary code for:

- Swap web3 for ethers and its wrapping class, taken from Web3API's monorepo

- ENS subdomain registration under "web3api.eth" domain
- ContentHash saving

- Domain owner resolution
- Domain contentHash resolution

Steps for testing:

- Connect wallet to ropsten
- Fill IPFS input
- Fill subdomain input
- Sign the 3 transaction prompts (register subdomain, set resolver, set content hash)

After a successful registration:

- Notice that when writing a new subdomain, the input field will show in red if the subdomain is already taken
- I didn't see where to output the content hash so I left that service unused for now but it's working, you can verify it with a simple invocation and console.log
